src/couch_replicator_api_wrap.erl

```diff
@@ -83,9 +83,16 @@ db_open(#httpdb{} = Db1, _Options, Create) ->
                     ok
                 end)
         end,
-        send_req(Db, [{method, head}],
-            fun(200, _, _) ->
-                {ok, Db};
+        send_req(Db, [{method, get}],
+            fun(200, _, {Props}) ->
+                case get_value(<<"update_seq">>, Props) of
```

Comment:
I know that, but assume that remote Source/Target is not a CouchDB. Then, this check ensures that remote is database by `update_seq` presence, runs replication and fill logs full of [out of sync](https://github.com/apache/couchdb-couch-replicator/blob/master/src/couch_replicator.erl#L777-L785) errors because `instance_start_time` also returns by `/_ensure_full_commit` endpoint what is [mandotarry](https://github.com/apache/couchdb-couch-replicator/blob/master/src/couch_replicator_api_wrap.erl#L184-L188).